runtime/lua/vim/lsp/protocol.lua

```diff
@@ -744,14 +744,15 @@ function protocol.make_client_capabilities()
       completion = {
         dynamicRegistration = false,
         completionItem = {
-          -- Until we can actually expand snippet, move cursor and allow for true snippet experience,
-          -- this should be disabled out of the box.
-          -- However, users can turn this back on if they have a snippet plugin.
-          snippetSupport = false,
+          snippetSupport = true,
           commitCharactersSupport = false,
           preselectSupport = false,
           deprecatedSupport = false,
           documentationFormat = { protocol.MarkupKind.Markdown, protocol.MarkupKind.PlainText },
+          resolveSupport = {
```

Comment:
[`lsp_compl`](https://github.com/mfussenegger/nvim-lsp-compl/blob/512abbbf40ca98e50b69f70a45e73496adfcc3ee/lua/lsp_compl.lua#L875) also includes `'detail'` and `'documentation'` here, but I'm unsure of doing that since there's actually no support for fetching that when resolving a completion item. If not present in the original completion request, those just default to an empty string.